### PR TITLE
Feature/cr 2840 aws cfn load balancer listeners

### PR DESCRIPTION
**Pull request by lirshindalman**

### CODE REVIEW

File: tests/knowledge/context/aws/test_load_balancer_listener.py [hunk 1]
```diff
@@ -1,32 +1,32 @@
...
+        self.assertEqual(distribution.listener_port, 80)
         self.assertEqual(distribution.default_action_type, 'forward')
         self.assertFalse(distribution.redirect_action_protocol)
         self.assertFalse(distribution.redirect_action_port)
-        self.assertEqual(distribution.get_cloud_resource_url(),
-                         'https://console.aws.amazon.com/ec2/v2/home?region=us-east-1#LoadBalancers:type=application')
 
-    @context(module_path="http")
-    def test_http_listener(self, ctx: AwsEnvironmentContext):
+    @context(module_path="https", test_options=TestOptions(run_terraform=False, run_cloudmapper=False))
```
Comment (Imanuel-Mizrahi) [inline tests/knowledge/context/aws/test_load_balancer_listener.py:20]: I guess we can remove the test_options from here, right?

File: tests/knowledge/context/base_context_test.py [hunk 2]
```diff
@@ -235,7 +235,7 @@ def _run_drift_detection_for_cloudformation(self, module_path: str, cfn_template
             if not os.path.isfile(cloudformation_template):
                 return
             if not os.path.isfile(account_data_for_drift_path + '.zip'):
-                raise Exception(f'missing account-data-for-drift-cfn.zip for {scenario_folder}')
+                raise Exception(f'missing account-data-for-drift-cfn2.zip for {scenario_folder}')
```
Comment (Imanuel-Mizrahi) [inline tests/knowledge/context/base_context_test.py:238]: please undo this change.

**Review by Imanuel-Mizrahi — CHANGES_REQUESTED** [review]

**Review by Imanuel-Mizrahi — APPROVED** [review]

**Review by Tomerva — APPROVED** [review]
